packages/make-styles/src/shorthands/borderLeft.ts

```diff
@@ -0,0 +1,30 @@
+import { BorderColorProperty, BorderStyleProperty, BorderWidthProperty } from 'csstype';
+import { MakeStyles, MakeStylesCSSValue } from '../types';
```

Comment:
```suggestion
import type { BorderColorProperty, BorderStyleProperty, BorderWidthProperty } from 'csstype';
import type { MakeStyles, MakeStylesCSSValue } from '../types';
```
I've seen this starting to become a thing in the repo, should we do it ?

Reply:
Well, there is no ESLint rule around it ¯\_(ツ)_/¯
But I will fix all imports 👍

Reply:
@dzearing had a series of PRs to change everything possible to `import type`, but some of them didn't get merged.  It looks like he also added `@typescript-eslint/eslint-plugin` rules [`consistent-type-imports`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-imports.md) and [`consistent-type-exports`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/consistent-type-exports.md) but didn't get around to enabling them here. 

The import rule has a clever way to avoid needing type information, which in theory should prevent it from being terribly expensive to run, but hard to say without measuring it. The exports rule *does* require type info and therefore could be slower.